display_list/display_list_builder.cc

```diff
@@ -229,6 +229,7 @@ void DisplayListBuilder::onSetImageFilter(const DlImageFilter* filter) {
         Push<SetSharedImageFilterOp>(0, 0, filter);
         break;
       }
+      case DlImageFilterType::kLocalMatrixFilter:
```

Comment:
This should be added to the previous set of cases in this switch so we add it with a shared pointer rather than converting to an opaque Skia object.

Reply:
(See SetSharedImageFilterOp implementation)